config/types/common.go

```diff
@@ -114,3 +119,41 @@ func getArgs(format, tagName string, e interface{}) []string {
 func getCliArgs(e interface{}) []string {
 	return getArgs("--%s=%v", "cli", e)
 }
+
+// Get returns the value for key, where key is an int or string and n is a
+// sequence node or mapping node, respectively.
+func get(n validate.AstNode, key interface{}) (validate.AstNode, error) {
```

Comment:
I'd prefer this to be named something a little more descriptive, since there are other functions here like `getCliArgs` that are entirely unrelated to `get`.